index.html

```diff
@@ -0,0 +1,131 @@
+            <p class="music-description">Harry Styles</p>
+          </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/lizzo-o9x-about-damn-time-f6q-344x344.jpg" alt="">
+            <span class="music-author">Lizzo </span>
+            <p class="music-description">About Damn Time </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/jack-harlow-i43-344x344.jpg" alt="">
+            <span class="music-author">Jack Harlow </span>
+            <p class="music-description">First Class </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/kate-bush-qv2-running-up-that-hill-1cx-344x344.jpg" alt="">
+            <span class="music-author">Kate Bush </span>
+            <p class="music-description">Running Up That Hill (A Deal With God)</p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/future-f8b-344x344.jpg" alt="">
+            <span class="music-author">Future Featuring Drake & Tems  </span>
+            <p class="music-description">Wait For U </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/drake-04g-344x344.jpg" alt="">
+            <span class="music-author">Drake Featuring 21 Savage </span>
+            <p class="music-description">Jimmy cooks</p>
+          </div>
+        </a>
+      </div>
+    </section>
+
+    <section id="featured-performers" class="featured-performers">
+      
+    </section>
+
+    <section id="partners" class="partners">
+      <h2>Partners</h2>
+      <div class="partners-container">
+        <img src="./images/pepsi-icon.svg" alt="pepsi-logo">
+        <img src="./images/espn-logo.svg" alt="espn-logo">
+        <img src="./images/puma-logo.svg" alt="puma-logo">
+        <img src="./images/mtv-live-seeklogo.com.svg" alt="mtv-logo">
+        <img src="./images/redbullenergydrink.svg" alt="red-bull">
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-poprax">&copy;PoPraX <sup>events</sup></div>
+      <ul>
+        <li><a href="#"><i class=" social-media-icons fa-brands fa-instagram"></i></a></li>
+        <li><a href="#"><i class="social-media-icons fa-brands fa-twitter"></i></a></li>
+        <li><a href="#"><i class="social-media-icons fa-brands fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="social-media-icons fa-brands fa-linkedin"></i></a></li>
+        <li><a href="#"><i class="social-media-icons fa-brands fa-youtube"></i></a></li>
+      </ul>
+      <div class="footer-content">
+      Terms of Use / Privacy Policy / Global Sites <br> 2022 PoPraX, Inc. All rights reserved. By continuing past this page, you agree to our terms of use.
+      </div>
+    </footer>
```

Comment:
- [ ] `required`
- [ ] just as mentioned on top I suggest you follow the original template here as well. please see the screenshots below

`your template`
![fot](https://user-images.githubusercontent.com/66230306/181394295-2aaf8e9e-b673-472d-88a5-bcb2c16ac8cd.png)

`his template`
![footer](https://user-images.githubusercontent.com/66230306/181394339-cf502649-ebf7-45a8-b83f-75df1d3c4e48.png)